### PR TITLE
Folder for 'collecting logs' documents

### PR DESCRIPTION
**Pull request by jovirkku**
Signed-off-by: Jorma Virkkunen <jorma.virkkunen@jolla.com>

### CODE REVIEW

**Comment by vigejolla** [comment]
I would prefer having the "moving files around" and "changing contents of the files" type of changes in separate commits. It would make reviewing easier. Now git doesn't realize that one of the files was moved, and shows instead that one file was deleted and another was added. If the commits were separated it would most likely show the actual differences.

File: Support/Help_Articles/Collecting_Logs/Collect_Logs_with_Logcat/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+The same about Android apps. Note that the command is different for different OS releases:
+* the 1st variant works up to OS release 4.4.0
+* the 2nd variant works from OS release 4.5.0 onwards
+
+```
+lxc-attach -n aliendalvik -- /system/bin/logcat > $MYHOME/android-logs.txt
+lxc-attach -n aliendalvik --lxcpath=/tmp/aliendalvik -- /system/bin/logcat > $MYHOME/android-logs.txt
+```
+The events in the Android system will now be printed to file ```android-logs.txt```. The command runs until you stop it with <ctrl>C.
```
Comment (vigejolla) [inline Support/Help_Articles/Collecting_Logs/Collect_Logs_with_Logcat/README.md:72]: The &lt; &gt; characters need to be escaped

**Comment by jovirkku** [comment]
This PR was reset. This improved one contains just the changes needed for the new folder structure: the new parent folder "Collecting Logs" and its README.md, and the two child subfolders Collecting_Logs_with_Logcat & Collecting_oFono_Logs. The headers of the READMEs of those subfolders were updated, of course. No other changes in them, yet.

**Comment by vigejolla** [comment]
> This PR was reset. This improved one contains just the changes needed for the new folder structure: the new parent folder "Collecting Logs" and its README.md, and the two child subfolders Collecting_Logs_with_Logcat & Collecting_oFono_Logs. The headers of the READMEs of those subfolders were updated, of course. No other changes in them, yet.

These look fine to me. Are you planning on reintroducing the other changes as a separate commit in this PR, or should I just merge this and wait for another PR for the other changes?

**Comment by jovirkku** [comment]
> > This PR was reset. This improved one contains just the changes needed for the new folder structure: the new parent folder "Collecting Logs" and its README.md, and the two child subfolders Collecting_Logs_with_Logcat & Collecting_oFono_Logs. The headers of the READMEs of those subfolders were updated, of course. No other changes in them, yet.
> 
> These look fine to me. Are you planning on reintroducing the other changes as a separate commit in this PR, or should I just merge this and wait for another PR for the other changes?

I would like to make one more commit for this PR to get the two existing READMEs fixed. The other log collection work (more documents) I would deliver in other PRs.

File: Support/Help_Articles/Collecting_Logs/Collect_Logs_with_Logcat/README.md (hunk 1 above)
Comment (jovirkku) [reply Support/Help_Articles/Collecting_Logs/Collect_Logs_with_Logcat/README.md]: Done.

**Review by jpwalden — APPROVED** [review]
LGTM

**Review by vigejolla — APPROVED** [review]

**Pull request merged**